gnd/src/test/java/com/google/android/gnd/persistence/local/LocalDataStoreTest.java

```diff
@@ -569,4 +576,24 @@ public void testGetOfflineAreas() {
         .test()
         .assertValue(ImmutableList.of(TEST_OFFLINE_AREA));
   }
+
+  @Test
+  public void testTimeFormat() {
+    Date originalDate = new Date();
+    String timeString = TimeResponse.convertToTimeFormat(originalDate);
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("HH:mm", Locale.getDefault());
+    LocalTime lt = LocalTime.parse(timeString, formatter);
+    String formattedTime = lt.format(formatter);
+    Truth.assertThat(timeString).isEqualTo(formattedTime);
+  }
+
+  @Test
+  public void testDateFormat() {
+    Date originalDate = new Date();
+    String dateString = DateResponse.convertToDateFormat(originalDate);
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd", Locale.getDefault());
+    LocalDate ld = LocalDate.parse(dateString, formatter);
+    String formattedDate = ld.format(formatter);
+    Truth.assertThat(dateString).isEqualTo(formattedDate);
+  }
```

Comment:
Please move these tests to a new `TimeResponseTest` class.

Reply:
updated in https://github.com/google/ground-android/pull/846/commits/f76997dfb4fb732af77602f52c869f31f18cd96b